src/main/java/org/commcare/formplayer/services/RestoreFactory.java

```diff
@@ -148,7 +149,7 @@ public void configure(String domain, String caseId, HqAuth auth) {
         this.setHqAuth(auth);
         this.hasRestored = false;
         this.configured = true;
-        sqLiteDB = new UserDB(domain, username, null);
+        sqLiteDB = new UserDB(domain, scrubbed_username, null);
```

Comment:
note: all other uses of `username` fall into the following buckets:

* logging
* sentry data
* as part of the device ID

Reply:
Is this going to break everyone's existing user dbs?

Reply:
no, `scrubbed_username` is the same value as `username` was prior to this change (https://github.com/dimagi/formplayer/pull/808/files#diff-7dff02876ec9dea6b13f04025dfe2b6231044a78026eb4c86ecd4553d398814fL715-R721)